release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -455,6 +455,34 @@ With this release, the etcd tuning parameters can be set to values that optimize
 * `Standard`
 * `Slower`
 
+[discrete]
+[id="ocp-4-16-unauthenticated-users-cluster-role-bindings_{context}"]
+=== Unauthenticated users were removed from some cluster role bindings
+
+With this release, unauthenticated users no longer have access to most cluster role bindings. Before {product-title} {product-version} unauthenticated users could access cluster role bindings. Changing this access for unauthenticated users adds an additional layer of security and does not impact most users. This change is for new clusters; previous clusters are not affected.
+
+There are use cases where allowing unauthenticated users specific cluster role bindings is desired. To grant unauthenticated users access to specific cluster role bindings that are necessary for certain features, see xref:../modules/unauthenticated-users-cluster-role-binding.adoc#modules-unauth-cluster-role-bindings[Adding unauthenticated groups to cluster role bindings].
+
+[IMPORTANT]
+====
+Always verify compliance with your organization's security standards when modifying unauthenticated access.
+====
+
+[discrete]
+[id="ocp-4-16-unauthenticated-users-system-webhook-role-binding_{context}"]
+=== Unauthenticated users were removed from the `system:webhook` role binding
+
+With this release, unauthenticated users no longer have access to the `system:webhook` role binding. Before {product-title} {product-version} unauthenticated users could access the `system:webhook` role binding. Changing this access for unauthenticated users adds an additional layer of security and does not impact most users. This change is for new clusters; previous clusters are not affected.
```

Comment:
```suggestion
With this release, unauthenticated users no longer have access to the `system:webhook` role binding. Before {product-title} {product-version} unauthenticated users could access the `system:webhook` role binding. Changing this access for unauthenticated users adds an additional layer of security and should only be enabled for users in need. This change is for new clusters; previous clusters are not affected.
```

I don't know how many do use that feature, but as long as it doesn't apply nearly everyone, it makes sense to make it opt-in.